docs/plugins/verifier/notation.md

```diff
@@ -23,9 +23,6 @@ spec:
       ca: # trust-store-type
         ca-certs: # name of the trustStore
           - <NAMESPACE>/<KEY MANAGEMENT PROVIDER NAME> # namespace/name of the key management provider CRD to include in this trustStore
-      tsa: # trust-store-type
```

Comment:
curious why we remove it

Reply:
To ensure CX understand that `tsa` config is optional in the `truststore` configuration.
Hi @FeynmanZhou, shall we keep this change or revert it

Reply:
reverted the changes in the first place

Reply:
@junczhu @binbin-li  I would suggest commenting two `tsa` fields as **Optional** and state that these fields are needed only when users enable timestamp countersignature verification. Otherwise, users might be confusing and even may fail to verify an image if they use `ca` cert only. 

For example, a new user can verify a signature successfully without `tsa` cert in a quick start. Timestamp requires additional cost (a timestamp server) to build trust. So it worth mentioning `tsa` is optional if we want to keep these fields. 

<img width="1018" alt="image" src="https://github.com/user-attachments/assets/91b09d13-a41c-485e-b0ca-b184afeb3368">

Reply:
Shall we mark `only required for timestamping feature` instead of `Optional` to make this more clear

Reply:
probably `optional, please remove if timestamp countersignature verification is not required`?

Reply:
@yizha1 Any suggestions?

Reply:
According to Notary Project spec, to enable timestamping validation, users need to configure the `tsa` type of trust stores in trust polices.

So, it could be users configure a `tsa` trust store, but not used in trust policies, then timestamping is still disabled.

To correctly enable timestamping, users need to configure the `tsa` trust stored and make sure it is used in trust policies.

I would suggest adding a comment in trust policy of Notation verifier, that to enable timestamping validation, users need to specify a valid `tsa` trust store.

Reply:
@yizha1 does [these lines](https://github.com/ratify-project/ratify-web/pull/118/files#:~:text=the%20%60verificationCertStores%60%20field.-,To%20use%20the%20timestamping%20feature%2C%20you%20need%20to%20configure%20the%20trust,to%20the%20%60tsa%2Dcerts%60%20certificate%20collection%20listed%20in%20the%20%60verificationCertStores%60%20field.,-%23%23%23%20CLI) helps explain the prerequisites?

Reply:
Added comments accordingly